### ISSUE TITLE
Release Moped v2.21

### ISSUE THREAD

**Issue by mddilley** [cityofaustin/atd-data-tech#19226]
- [x] bump Moped editor `package.json`  version to `v2.21.0` @mddilley 
- [x] schedule release party - @mddilley 
- [x] update [QA script](https://docs.google.com/spreadsheets/d/1n_O6MLh9cwwPf57HUM394Ea-z9uuoEV1-QW4axNZXLE/edit#gid=1099176671) as needed - @mddilley 
- [x] advise users of downtime using [template](https://docs.google.com/document/d/1hpBIx41mttSF6-XgeFuxnJPsK7dfpok97O08oytThnc/edit#heading=h.51j00pz8giuz) - @mddilley
- [x] merge open PRs - @mddilley
- [x] propose + vote on release names (water features) - @chiaberry
- [x] cut release branch - @chiaberry 
- [x] do testing - @mddilley 
- [x] polish release notes - @chiaberry 
- [x] let users know maintenance is starting - @mddilley
- [x] snap database in RDS - @mddilley
- [x] ~Did we add or remove columns from DB views [that the Power BI dataflow depends on](https://atd-dts.gitbook.io/moped-documentation/product-management/updating-the-arcgis-online-components-database-view)?~ Nope!
   ~- [ ] Let the Data Science team know~
   ~- [ ] Do [full replace](https://github.com/cityofaustin/atd-moped/tree/main/moped-etl/arcgis) of AGOL dataset if needed~
- [x] merge RC to prod - @mddilley
- [x] graphql-engine version bumps ([releases & tags](https://github.com/hasura/graphql-engine/releases)) - @mddilley
  - [x] update local dev docker compose to latest `graphql-engine` (`2.44.0`)
  - [x] update staging task definition to latest `graphql-engine` release (`2.44.0`)
  - [x] ~update production task definition to previous staging release version + any patches (`version`)~
  - [x] deploy task definition for staging (TD 48)
  - [x] ~deploy task definition for production (TD )~
- [x] let users know maintenance is over using [template](https://docs.google.com/document/d/1hpBIx41mttSF6-XgeFuxnJPsK7dfpok97O08oytThnc/edit#heading=h.51j00pz8giuz) - @mddilley
- [x] create issue for `v2.22.0` release - @mddilley
- [x] update [feature list](https://app.gitbook.com/o/-LzDQOVGhTudbKRDGpUA/s/-MIQvl_rKnZ_-wHRdp4J/product-management/features) with any new major features @mddilley (No update needed)
- [x] send release notes to MUG using [template](https://docs.google.com/document/d/1hpBIx41mttSF6-XgeFuxnJPsK7dfpok97O08oytThnc/edit#heading=h.jf63bjq5syy7) (including reminder to refresh the page) - @mddilley 
- [x] update staff accounts if there are any bounced emails - @mddilley

**Comment by mddilley** [comment]
Last release https://github.com/cityofaustin/atd-data-tech/issues/19035

**Comment by frankhereford** [comment]
Just now, I see that @mddilley had reassigned my normal tasks to himself because I'm going to be out of town, so first off -- thank you Mike!! I appreciate you being so thoughtful and on top of everything. 

And before I realized that, I got some of this squared away early. As of the time of writing:

* ✅ RDS snapshot is snapped
* ✅ Task definition for staging is updated for the latest `graphql-engine` `2.44.0`
* ✅ Staging cluster has been updated to the above task definition.

* [ ] Remaining: Bump local dev stacy to `2.44.0` in `main` after the mergin' is done

Thanks y'all! See you tomorrow. 🙏🚀🙏🛵🙏🙏🛵 🚀🚢 💨
